serverless/dextr/openfaas/index.sh

```diff
@@ -0,0 +1,2 @@
+. /opt/intel/openvino/bin/setupvars.sh > /dev/null
```

Comment:
Codacy found an issue: [Tips depend on target shell and yours is unknown. Add a shebang.](https://app.codacy.com/manual/cvat/cvat/pullRequest?prid=5729179)